lib/resolver/homebrew-file.js

```diff
@@ -2,7 +2,7 @@ import relativeFile from './relative-file.js';
 
 export default function ({ path, target }) {
   return [
-    relativeFile({ path, target }),
-    '{BASE_URL}/Homebrew/homebrew-core/blob/master/Formula/' + target,
+    relativeFile({ path, target, baseUrl: 'https://github.com' }),
```

Comment:
Oh, I should have explained more in [my previous comment](https://github.com/OctoLinker/browser-extension/pull/174#discussion_r77091612). I don't think `relativeFile` needs to change, only that the fallback URL on the line below this one should always start with https://github.com/Homebrew/homebrew-core. For example, consider the case where you have a custom [Homebrew tap](https://github.com/Homebrew/brew/blob/6622b565ee38de1cd97f5c7b20fa62b481a1ec9b/share/doc/homebrew/brew-tap.md) hosted at https://enterprise-github.example.com/user/repo:

If https://enterprise-github.example.com/user/repo/blob/abcdef/custom-application.rb has a dependency on `custom-library`, it should link to https://enterprise-github.example.com/user/repo/blob/abcdef/custom-library.rb rather than https://github.com/Homebrew/homebrew-core/blob/abcdef/custom-library.rb.

Does that make more sense? If you'd like, I can push up a branch that has my preferred version of this PR.

(See https://github.com/OctoLinker/browser-extension/pull/134 for more context about the homebrew-file resolver)